src/platform/unittest/TlsTest.cpp

```diff
@@ -273,6 +273,8 @@ struct TlsTest : public ::testing::TestWithParam<bool>
 
         CXPLAT_TLS_PROCESS_STATE State;
 
+        bool BufferKeyChecked;
```

Comment:
There should be some comment around this and its usage to explain it. It's not clear why you only check a key once.